src/server/router.js

```diff
@@ -4,8 +4,12 @@ let express = require('express');
 let controller = require('./controller');
 let router = express.Router();
 
+// router.get('/api/homepage', memcacheMiddleWare, controller.getHomepage);
```

Comment:
Remove commented code, or add a TODO comment explaining why it is commented

Reply:
done